### PR TITLE
Adding Edge Query X Adapter (with right md file)

### PR DESCRIPTION
**Pull request by edgequery**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ X ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Creating a new Adapter for Edge Query X format.

- test parameters for validating bids
```
                {
                    bidder: "edgequeryx",
                    params: {
                        accountId: "test",
                        widgetId: "test"
                    }
                }
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer : olivier@edgequery.com
- [ X ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
https://github.com/prebid/prebid.github.io/pull/1962

### CODE REVIEW

**Comment by smenzer** [comment]
@edgequery just an fyi that you don't need to submit a new PR every time you make changes. If you just push new commits to your branch (in your fork) it will update the existing PR. This way the history of the PR is maintained and the reviewer has continuity on the updates they requested.

**Comment by edgequery** [comment]
Ok, now all must be correct, tests are passed with more than 80%

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "prebid.js",
-  "version": "3.18.0-pre",
+  "version": "3.20.0-pre",
```
Comment (aleksatr) [inline package-lock.json:3]: you should not modify `package-lock.json` as part of the adapter submission PR, please revert changes you made to this file.

File: test/spec/modules/edgequeryxBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import {
```
Comment (aleksatr) [inline test/spec/modules/edgequeryxBidAdapter_spec.js:1]: any specific reason why you use this multi-line format for imports here, while on lines 13 and 14 you're inlining them?

**Comment by aleksatr** [comment]
hey @edgequery , I tested your adapter with params from `.md` file, test endpoint does return the bid, but it's invalid (request ID doesn't match):

```
WARNING: Bidder edgequeryx made bid for unknown request ID: abcd1234. Ignoring.
```
I guess that's not crucial for you, so if you would fix comments I added (mainly reverting the changes to package-lock.json), we can proceed and merge this.

**Comment by aleksatr** [comment]
LGTM

**Pull request merged**